Server/API/rooms/guess/postGuess.go

```diff
@@ -0,0 +1,51 @@
+package guess
+
+import (
+	"context"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	roomsScripts "ishtaloo.io/DB/Scripts/rooms"
+	cookieUtils "ishtaloo.io/Utils/cookie"
+)
+
+func postGuess(c echo.Context) (err error) {
+	roomId := c.Param("roomId")
```

Comment:
the roomId will most likely be sent in the requst's body

Reply:
Why?

Reply:
will do